components/validation/src/main/java/org/wso2/carbon/identity/x509Certificate/validation/validator/RevocationValidator.java

```diff
@@ -18,8 +18,8 @@
 
 package org.wso2.carbon.identity.x509Certificate.validation.validator;
 
-import org.wso2.carbon.identity.x509Certificate.validation.CertificateValidationException;
-import org.wso2.carbon.identity.x509Certificate.validation.RevocationStatus;
+import org.wso2.carbon.identity.x509Certificate.validation.exception.CertificateValidationException;
+import org.wso2.carbon.identity.x509Certificate.validation.model.RevocationStatus;
```

Comment:
Same comment as the `RevocationValidationManager`. Better to avoid changing the extension point.

Reply:
fixed with https://github.com/wso2-extensions/identity-x509-commons/pull/56/commits/ee5b98f65b697623ef65f7d1a6ee245e1c9f982d